File: Homework_2/access_level.py
def access_level(rights: list, minPermission: int):
    output: str = ''
    for right in rights:
        if right >= minPermission:
            output += 'A'
        else:
            output += 'D'
    return output

if __name__ == '__main__':
    print(access_level([0,1,2,3,4,5], 2))
    print(access_level([5,3,2,10,0], 20))
    print(access_level([], 20))
    print(access_level([34,78,9,52,11,1], 49))
    

File: Homework_2/access_level.c
#include <stdio.h>
#include <stdlib.h>

void accessLevel(int* rights, int minPermission) {
    int length = sizeof(rights) - sizeof(rights[0]);

    char* output = (char*) malloc(length * sizeof(char));

    for (int i = 0; i < length; i++) {
        if (rights[i] >= minPermission) {
            
        }
    }

} 

int main() {
    int rights[] = {1, 2, 3};

    accessLevel(rights, 3);
    return 0;
}